apps/portal/src/components/documentation/InstallationGuideSection.tsx

```
@@ -462,6 +487,30 @@ cd qarote`}
                     __html: t("installGuide.quickStart.binary.step2Note"),
                   }}
                 />
+                <div className="bg-muted rounded-lg p-4 mt-2 space-y-2">
+                  <p className="text-sm font-medium">
+                    {t("installGuide.quickStart.binary.setupWizardTitle")}
+                  </p>
+                  <ul className="text-sm text-muted-foreground list-disc list-inside space-y-1">
+                    <li
+                      dangerouslySetInnerHTML={{
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)